test/fixtures/test-runner/aborts/wait-for-abort-helper.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+  waitForAbort: function ({ testNumber, signal }) {
```

Comment:
why not `await Promise.race(once(signal, 'abort'), setTimeout(1000)`?

Reply:
Because we wait for the `afterEach` in the test runner...

https://github.com/nodejs/node/blob/f5f36b601416084cf294bbc9a7f0cd2deaea8201/lib/internal/test_runner/test.js#L589-L590

Reply:
so doing await will always fail...